lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx

```diff
@@ -113,10 +117,14 @@ class HelpPanelItem extends React.Component {
           itemString={this.props.content.itemName}
           paneMode={this.props.content.paneMode}
           markdownContent={this.props.content.markdownText}
-          videoUrl={useTranslationIfExists(this.props.content.videoUrl)}
-          placeholderImage={useTranslationIfExists(
-            this.props.content.placeholderImage
-          )}
+          videoUrl={
+            this.props.content.videoUrl &&
+            useTranslationIfExists(this.props.content.videoUrl, i18n)
+          }
+          placeholderImage={
+            this.props.content.videoUrl &&
+            useTranslationIfExists(this.props.content.placeholderImage, i18n)
```

Comment:
this should actually check if `placeholderImage` is defined

Reply:
Done